docker/Makefile

```diff
@@ -0,0 +1,131 @@
+
```

Comment:
Let me know if I need to comment this. (Maybe you just accept this as students will probably do not use it)